packages/web3-core/types/index.d.ts

```diff
@@ -18,6 +18,7 @@
  */
 
 import * as net from 'net';
+import BN = require('bn.js');
```

Comment:
Could you change it to: ``import BN from 'bn.js'``

Reply:
Hello!

I can do it, no problem. But according to TypeScript [handbook](https://www.typescriptlang.org/docs/handbook/modules.html) 
>When exporting a module using export =, TypeScript-specific import module = require("module") must be used to import the module.

BN is exported using `export =`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/bn.js/index.d.ts#L584

What do you think?

Reply:
Great, thanks for referencing the documentation! Yes, then we will use your way ;-)